### PR TITLE
Add "Badges 4 Life" through a config var

### PR DESCRIPTION
**Pull request by daredoes**
This pull request will allow a list of emails for people who deserve badges for life to MAGFest events to receive them without human intervention. 

The emails must be loaded securely to the config settings for the privacy of individuals who receive badges for life.

Who decides the list is not up to me. 

Personal Opinions:
My suggestion would be the acting executive director of MAGFest, but I leave this up to the powers that be.

Also, Brendan Becker needs to be in this list.

I haven't tested this unfortunately, but its a straight copy of the `_use_promo_code` function above it. 

I read through the logic on the forms, and this should work. 

It ain't easy to setup Uber, but I hope this helps.

### CODE REVIEW

**Comment by EliAndrewC** [comment]
I appreciate the PR, but after a lot of discussion, the BoD decided to not implement a formal list like this.  Note that this is different from saying that there are no such people.  For example, former employees of MAGFest Inc have been told that the MAGFest office will create a complementary badge/ticket for them at any MAGFest event upon request.  The same goes for people like Brendan.

However, creating a format "badge for lifers" category was something we didn't like, because it creates a sharp divide between haves and have-nots.  For instance, there are often people who we want to offer complementary badges for a number of events over a few years, but who probably shouldn't be on a "badge for life" list.  By authorizing people in leadership such as office and BoD members to be allowed to create complementary badges upon request, we are able to satisfy the underlying requirement without needing to create and maintain a formal list with official selection criteria and analysis, etc.

That being said, thanks for pitching in - even if we're closing this rather than merging it, the sentiment is much appreciated.

**Comment by daredoes** [comment]
I understand where you're coming from, yet I would still implore you to reconsider the codification of emails that can receive at least one badge to all MAGFest events.

Something I've always found beautiful about MAGFest events is how Uber acts as a rigid structure guiding what can and cannot not occur. 

What should not occur is someone who has been given complimentary badges in a seemingly-for-life capacity missing out on a badge because of preventable human error. Human error could be busyness, or even the changing of the guard at the BoD, Office, or even STOPS level.

I do not believe the list should be used liberally, but I do believe it is important that one is used.

From my experience, when someone feels the impact of this human error it creates a feeling of self-worthlessness. The sentiment I feel, and I do not believe I am alone in feeling, is something like, "For an event that considers giving badges away akin to giving away candy, I don't feel they valued my contributions."

From a more personal experience, I didn't come to MAGWest 3 because buying a badge for the event made me feel worthless and afraid. Afraid because I know some people in STOPS don't like my existence 😇. 

Someone offered to create me a badge in the system, but I declined because, among other reasons, I didn't want them to take flak for giving me a badge. I wrote this PR because the thought of being rejected, no actually the thought of it even being a question, when asking for a badge to future MAGWests leaves me too anxious to feel asking has any positive value in my life.

To summarize: reminding humans of prior accomplishments in a gamble to receive a complimentary badge feels needy and subject to whim. Some humans have done enough for MAGFest that they should not need to go through this. Uber should ensure their ability to attend, human-free.

I do not believe this list needs "Official Selection Criteria" beyond the approval of the acting executive director. If we cannot trust the executive director to use their best judgement on adding members to the list, how can we trust them with the heart, soul, and bank account of MAGFest?

However things turn out, this PR will be around waiting for the day it should be merged.

Thanks for hearing me out.
